lms/templates/bookmarks/bookmarks_list.underscore

```diff
@@ -0,0 +1,43 @@
+<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
+<h2 class="bookmarks-results-header"><%= gettext("My Bookmarks") %></h2>
+
+<% if (bookmarks.length) { %>
+
+    <div class='bookmarks-results-list'>
+        <% _.each(bookmarks, function(bookmark, index) { %>
+             <a class="bookmarks-results-list-item" href="<%= bookmarkUrl(bookmark.get('course_id'), bookmark.get('usage_id')) %>" aria-describedby="bookmark-type-<%= index %> bookmark-date-<%= index %>">
+                <div class="list-item-content">
+                    <div class="list-item-left-section">
+                        <h3 class="list-item-breadcrumbtrail"> <%= breadcrumbTrail(bookmark.get('path'), bookmark.get('display_name')) %> </h3>
+                        <p id="bookmark-date-<%= index %>" class="list-item-date"> <%= gettext("Bookmarked on") %> <%= userFriendlyDate(bookmark.get('created')) %> </p>
+                    </div>
+
+                    <p id="bookmark-type-<%= index %>" class="list-item-right-section">
+                        <span aria-hidden="true">View</span>
+                        <i class="icon fa fa-arrow-right" aria-hidden="true"></i>
+                    </p>
+                </div>
+            </a>
+        <% }); %>
+    </div>
+
+<% } else {%>
+
+    <div class="bookmarks-empty" tabindex="0">
+        <div class="bookmarks-empty-header">
+            <i class="icon fa fa-bookmark-o bookmarks-empty-header-icon" aria-hidden="true"></i>
+            You have not bookmarked any courseware pages yet.
+            <br>
+        </div>
+        <div  class="bookmarks-empty-detail">
+            <span class="bookmarks-empty-detail-title">
+                <%= gettext("Use bookmarks to help you easily return to courseware pages. To bookmark a page, select Bookmark in the upper right corner of that page. To see a list of all your bookmarks, select Bookmarks in the upper left corner of any courseware page.") %>
+            </span>
+            <span class="bookmarks-empty-detail-bookmark-button" aria-hidden="true">
+                <i class="icon fa fa-bookmark-o bookmarks-empty-header-icon"></i>
+                Bookmarks
```

Comment:
@marcotuts I think what you mean is removing the little "bookmarks" button. If that's the case, then yes, we can remove it because the text itself is enough explanation.